README.md

```diff
@@ -0,0 +1,234 @@
+  <img src="doctors Icon.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Doctors Reservations Back-End</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Doctors Reservations Backend <a name="about-project"></a>
+
+
+**Doctors reservations Backend** is a back-end implementation of the front-end application that interacts with the back-end application to add doctors, logins, and reservations.
+
+## 🛠 Built With <a name="built-with"></a>
+- Rails
+- Ruby
+- Rspec
+- Api Documentation
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Implement the reservations,  using React and Ruby on rails**
+- **Demonstrate ability to apply best practices of communication for resolving teamwork challenges.**
+- **Add new doctors and reservations and using API REST to deploying in the Front-end Repository**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## Doctors Reservations Front-End
+
+[Front End App](https://github.com/JonahKayizzi/doctors-reservations-frontend.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+ gem install rails
+ bundle install
+ rails db:create
+ rails db:migrate
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/JonahKayizzi/Doctors-Reservations-Backend
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd my-project
+  gem install
+  bundle install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  rails server or rails s
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  rspec .
+```
```

Comment:
- [ ] If I run the tests according to your instructions, I get this:

![image](https://user-images.githubusercontent.com/57421823/223498850-d332a379-dd58-40dd-b6b0-84d4cdbdf461.png)

## Explanation

This is because the majority of your tests are swagger tests.

## Tips and extra help

Just be more specific on your command that is featured on your readme instructions:

```bash
rspec ./spec/models
rspec ./spec/integration
```

That way, all your tests will pass for your users. 💪 

- [ ] Also, regarding `rspec ./spec/models`, there is one test that is not passing on my end:

![image](https://user-images.githubusercontent.com/57421823/223499895-75dcffed-e8c3-4a0d-b1f2-2963cc5de9cd.png)

There seems to be an issue with the date.

Kindly fix this tests to ensure your app's integrity. 👍